e2e-tests/manifest

```diff
@@ -0,0 +1,88 @@
+#!/usr/bin/env bash
+
+set -ex
+
+# https://v0-19-x.sdk.operatorframework.io/docs/install-operator-sdk/
+#operator-sdk generate k8s
+operator-sdk generate crds
+
+crd_ver=${1}
+if [ "${crd_ver}" == "" ]; then
+	crd_ver=$(yq r -d0 "deploy/crd.yaml" "spec.versions[-1].name")
+fi
+
+wip_crd=$(mktemp)
+yq r -d0 "deploy/crd.yaml" \
```

Comment:
In order to avoid context errors we need to use pull pathes, not the relative ones